### ISSUE TITLE
Turn on TypeScript `useUnknownInCatchVariables` option

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#779]
In the upcoming TypeScript 4.4, there's an option for having catch clauses be treated as unknown: https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables

This will eliminate the need for declaring the type on all catch clauses introduced in: https://github.com/pixiebrix/pixiebrix-extension/pull/772

Blocked by: https://github.com/pixiebrix/pixiebrix-extension/issues/778

Low priority since the linting rule doesn't add much more work
